### DISCUSSION TITLE
Graylevels for raw tilt series

### DISCUSSION THREAD

**Discussion by asarnow** [nextpyp/.github#90] [Q&A] [answered]
I was wondering how the graylevel / contrasts are selected for the raw tilt view and when excluding tilts. If a tilt is visually dark around the region of interest should it be excluded?

**Comment by abartesaghi** [comment, marked as answer]
Probably yes. The contrast is adjusted for each tilt individually using ImageMagic's command: `convert -contrast-stretch 1%x2%`.
See their [documentation](https://imagemagick.org/Usage/color_mods/#contrast-stretch) for more details.
